libs/entities/occupancy_rule.py

```diff
@@ -0,0 +1,14 @@
+from datetime import time, datetime
+
+class OccupancyRule:
+
+    def __init__(self, json_value: dict):
+        self.days = json_value["days"]
+        start_time = json_value["start_hour"].split(":")
+        self.start_hour = time(int(start_time[0]), int(start_time[1]))
+        finish_time = json_value["finish_hour"].split(":")
+        self.finish_hour = time(int(finish_time[0]), int(finish_time[1]))
+        self.occupancy_threshold = json_value["max_occupancy"]
```

Comment:
Maybe we can include the validation  `start_time < end_time`.

Reply:
I think that for this version we can split the range into two

Reply:
Yeah we agreed with FE that time ranges are only valid during the day. I have just added the validations with the endpoints